ui/src/views/Status.vue

```diff
@@ -44,11 +44,37 @@
         <NsInfoCard
           light
           :title="status.instance || '-'"
-          :description="$t('status.app_instance')"
+          :description="
+            config && config.host ? $t('status.app_instance') : $t('status.not_configured')
```

Comment:
```suggestion
            config && config.host ? config.host : $t('status.not_configured')
```
Why not showing actual URL instead of "App instance"?

Reply:
Matter of taste, back to the original code :D